components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationUserResidentResolverServiceImpl.java

```diff
@@ -225,6 +226,25 @@ the accessed organization (lower level)
         return basicOrganizationList;
     }
 
+    public User getUserFromResidentOrgId(String userId, String userResidentOrganizationId)
+            throws OrganizationManagementException {
+
+        User user = null;
+        try {
+            String associatedTenantDomainForOrg = resolveTenantDomainForOrg(userResidentOrganizationId);
+            if (associatedTenantDomainForOrg != null) {
```

Comment:
Can we use the StringUtils here?

Reply:
Resolved in [00398aa](https://github.com/wso2-extensions/identity-organization-management-core/pull/61/commits/00398aa7023f8c9db79ab89f028e8267f73eb519)